src/Illuminate/Database/Console/Migrations/MigrateMakeCommand.php

```diff
@@ -132,4 +136,26 @@ protected function getMigrationPath()
 
         return parent::getMigrationPath();
     }
+
+    /**
+     * Get the columns from the columns option and format them
+     *
+     * @return array
+     */
+    protected function getColumns()
+    {
+        $columns = [];
+        foreach ($this->input->getOption('columns') ?? [] as $column) {
+            $values = explode(':', $column);
+            if (count($values) < 2) {
+                $this->error("$column is invalid, the format is name:type");
+                abort(0);
+            }
+
+
```

Comment:
i think you should remove trailing spaces

Reply:
I've removed the extra empty line